ReduxCore/inc/fields/typography/field_typography.js

```diff
@@ -349,20 +496,25 @@
         //console.log(color);
 
         //var output = family;
+        
+        //add
+        if(active){
```

Comment:
This won't work.  the google and detail vars are loaded in this if/then block, and instances of it fall out of scope when the block ends.